main.js

```diff
@@ -0,0 +1,157 @@
+// 1 //
+
+const validity = function (value) {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return String(value);
+};
+
+const autoTeam = function (
+  teamName,
+  driverName,
+  yearBirth,
+  brandCar,
+  sponsor,
+  driverAcess
+) {
+  const newAutoTeam = {
+    teamName: validity(teamName),
+    driverName: validity(driverName),
+    yearBirth: validity(yearBirth),
+    brandCar: validity(brandCar),
+    sponsor: validity(sponsor),
+    driverAcess: driverAcess,
+  };
+  return newAutoTeam;
+};
+
+autoTeam("Valhalla", "Ulrih", 1980, "Volvo", "Norway Ragnarok", "yes");
+
+// 2 //
+
+const empty = "empty__field";
+
+const newUser = function () {
+  const user = {
+    login: ``,
+    password: ``,
+    city: ``,
+    country: ``,
+    maleOrFemale: ``,
+    age: ``,
+  };
+
+  for (const [key] of Object.entries(user)) {
+    const answer = prompt([key]);
+    if (answer === null || answer === ``) user[key] = empty;
+    else {
+      user[key] = answer;
+    }
+  }
+  return user;
+};
+const userCreated = newUser();
+console.log(userCreated);
+
+// 3 //
+
+const changeDATA = function (Object, key, value) {
```

Comment:
Strange name for the function, try to use one naming convention at a time.
`Object` param can be without uppercase